docs/en/advanced-features/grpc-tls.md

```diff
@@ -0,0 +1,36 @@
+# Support Transport Layer Security (TLS)
+Transport Layer Security (TLS) is a very common security way when transport data through Internet.
+In some use cases, end users report the background:
+
+> Target(under monitoring) applications are in a region, which also named VPC,
+at the same time, the SkyWalking backend is in another region (VPC).
+>
+> Because of that, security requirement is very obvious.
```

Comment:
```suggestion
```

Is this from some existing doc? In today(2023) arch, even inside VPC, the TLS is being considered necessary, such as ZTA.

Reply:
Yes, I have re-write from java agent.

Reply:
Maybe Java side should be polished as well. TLS has been required much more than ever before.